Using InstantSearch in typescript requires to install places.js since v4.52.0

### 🐛 Current behavior

When upgrading `instantsearch.js` to v.52 > with typescript, the `places.js` module is not found:

```
Cannot find module 'places.js' or its corresponding type declarations.
```

Similar to this issue: #4911 but with `places.js`.

When I add `places.js` in my packages, it works.


### 🔍 Steps to reproduce

1. open new directory
In the terminal: 
```
$ npm init -y
$ npx tsc --init
$ yarn add @types/node --save-dev
$ yarn add instantsearch.js 
```
2. Create index.ts file with the following content:
```ts
import instantsearch from "instantsearch.js";

console.log(instantsearch)
```
3. run `npx tsc index.ts`

### Live reproduction

https://github.com/bidoubiwa/instantsearch-v4.52.0-ts

### 💭 Expected behavior

No errors, or maybe add it in the peerDependencies

### Package version

instantsearch.js 5.53.0

### Operating system

macOs 13.2.1

### Browser

/

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

--- a/packages/instantsearch.js/src/widgets/places/places.ts
+++ b/packages/instantsearch.js/src/widgets/places/places.ts
@@ -1,5 +1,6 @@
-/** @ts-ignore */
 import type { WidgetFactory, WidgetRenderState } from '../../types';
+/* Places.js is an optional dependency, no error should be reported if the package is missing */
+/** @ts-ignore */
 import type * as Places from 'places.js';
 
 // using the type like this requires only one ts-ignore

